bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-catalog",
-  "version": "0.0.68",
+  "version": "0.0.69",
```

Comment:
We're switching to `v3.9.0-alpha.0` (1, 2, 3...) for releases as in common

https://github.com/openshift/origin-web-common/releases/tag/v3.9.0-alpha.0

Reply:
OK, thanks. I'll update